src/apollo/index.ts

```diff
@@ -1,17 +1,32 @@
 import ApolloClient from 'apollo-client';
 import { createHttpLink } from 'apollo-link-http';
 import { InMemoryCache } from 'apollo-cache-inmemory';
+import { setContext } from "apollo-link-context";
 
 const url = process.env.VUE_APP_GRAPHQL;
 
 const link = createHttpLink({
   uri: url + '/graphql',
 });
 
+const authLink = setContext((_, { headers }) => {
+  // get the authentication token from local storage if it exists
+  const token = localStorage.getItem('repoAdoptToken');
```

Comment:
I would prefer if you used the store above the local storage. Local storage is more vulnerable compared to the store